src/polygons.jl

```diff
@@ -225,5 +280,6 @@ function Base.read(io::IO, ::Type{PolygonZ})
     points = _readpoints(io, numpoints)
     zrange, zvalues = _readz(io, numpoints)
     mrange, measures = _readm(io, numpoints)
-    PolygonZ(box, parts, points, zrange, zvalues, mrange, measures)
+    indexcache = Vector{Bool}[]
```

Comment:
Why is `indexcache` a `Vector{Bool}[]` here, but a `Vector{Int}[]` in the struct definition?  This seems to be in all the `read` methods.

Reply:
Yeah I just left it in there by accident from when was Bool, already deleted :)